### ISSUE TITLE
Occasional errors with getaddrinfo

### ISSUE THREAD

**Issue by dhofverberg** [spaam/svtplay-dl#241]
Quite often I get errors when downloading from TV3 Play, and sometimes SVT Play as well. The errors are never consistent, so when trying again it never fails on the same place. This is how it looks like:

INFO: Outfile: revenge-avsnitt-22-tv3-play.srt
INFO: Selected to download hls, bitrate: 2796
INFO: Outfile: revenge-avsnitt-22-tv3-play.ts
[064/248][===========..................................] ETA: 0:08:45ERROR: Cant
 find that page: [Errno 11004] getaddrinfo failed

Example URL: http://www.tv3play.se/program/revenge/619577?autostart=true

The problem does not appear to be specific to select episodes or shows, as it happens occasionally on all videos I tried with. I have never been able to reproduce the problem in a consistent manner, as it seems to be random whether it happens or not.

Any ideas why this is, or if it's possible to do something about it?

**Comment by Sopor-** [comment]
Sounds like a bad Internet connection. I have already reported a similar issue but it was on SVTPlay.

**Comment by dentarg** [comment]
Maybe svtplay-dl should catch getaddrinfo and print useful debugging information when it happens

**Comment by olof** [comment]
Yeah, maybe. But would be useful with a full stacktrace. When this happens, is it reproducible? If so, could you rerun with --verbose? Also, this was prior to the requests rewrite I think, how is it working now?
